share/doc/homebrew/Checksum_Deprecation.md

```diff
@@ -0,0 +1,22 @@
+During early 2015 Homebrew started the process of deprecating _SHA1_ for package
+integrity verification. Since then every formulae under the Homebrew organisation
+has since been moved onto _SHA256_ verification; this includes both source packages
+and our precompiled packages (bottles).
+
+We also stopped supporting _MD5_ entirely. It was removed from core formulae towards
+the end of 2012 but till 2015 if you tried to install a formula still using an
+_MD5_ checksum Homebrew wouldn't actively stop you. Since that point it has.
+
+On _SHA1_ we added a `brew audit` check that flags _SHA1_ checksums as deprecated
+and requests that you use _SHA256_.
+
+From March 20th 2016 we've stepped up the visibility of that notification & you'll start
+seeing deprecation warnings when installing _SHA1_-validated formula.
+If you see these please consider reporting it to where the formula originated.
+
+We're targeting **the end of September 2016** for _SHA1_ support removal,
+19 months after we started warning people to move away from it for verification.
+This will be enforced in the same way _MD5_ is today.
```

Comment:
Doesn't our SHA1 support match our MD5 support whenever it's moved to `compat`? I can't quite remember but did we actually break the DSL for `md5` or `odie` or what?